app/models/product.rb

```diff
@@ -1,5 +1,6 @@
 class Product < ApplicationRecord
   has_and_belongs_to_many :categories
+  #should it be has many categories through that one table?
```

Comment:
I think our original thinking was to make this one a join table which would I think need the "has_and_belongs_to_many". If we want to change it to an indirect table, we can change this syntax to the "through" thing from before!